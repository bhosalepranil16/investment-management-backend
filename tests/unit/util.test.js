const { generateOTP, errorHandler } = require("../../util/index");

describe("util.js", () => {
  it("Test generateOTP", () => {
    expect(generateOTP("error message")).toMatchSnapshot();
  });

  it("Test errorHandler", () => {
    expect(errorHandler("error message")).toMatchSnapshot();
  });
});
