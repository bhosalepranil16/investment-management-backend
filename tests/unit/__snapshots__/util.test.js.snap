// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`util.js Test errorHandler 1`] = `
{
  "msg": "error message",
}
`;

exports[`util.js Test generateOTP 1`] = `"510615"`;
